common.sh

```diff
@@ -199,7 +199,9 @@ create_pg()
     if [ ${ENABLE_PLACEMENT_GROUP} -eq 0 ]; then
         return 0
     fi
-    PLACEMENT_GROUP="slave-pg-$RANDOM"
+    #Month - Day - Year - Hour - Minute - Second
+    date_time=$(date +'%m-%d-%Y-%H-%M-%S')
```

Comment:
If the jobs are run in parallel, would there be an issue with the timings?

Reply:
`BUILD_NUMBER` could be the same for parallel jobs. For Jenkins, `BUILD_NUMBER` is the same for all parallel stages within the same build.